devtools/datasette/fly/run.sh

```diff
@@ -4,7 +4,8 @@ set -eux
 shopt -s nullglob
 
 find /data/ -name '*.sqlite' -delete
+ls
 mv all_dbs.tar.zst /data
 zstd -f -d /data/all_dbs.tar.zst -o /data/all_dbs.tar
 tar -xf /data/all_dbs.tar --directory /data
-datasette serve --host 0.0.0.0 /data/pudl.sqlite /data/ferc*.sqlite /data/censusdp1tract.sqlite --cors --inspect-file inspect-data.json --metadata metadata.yml --setting sql_time_limit_ms 5000 --port $PORT
+datasette serve --host 0.0.0.0 ${DATABASES} --cors --inspect-file inspect-data.json --metadata metadata.yml --setting sql_time_limit_ms 5000 --port $PORT
```

Comment:
This lets us configure the databases & their ordering via env var in Python. Sort of janky, but 🤷